src_assets/common/assets/web/footer.html

```diff
@@ -0,0 +1,15 @@
+<script src="/scripts/main.js"></script>
+
+<!-- Discord WidgetBot Crate-->
+<script src="https://cdn.jsdelivr.net/npm/@widgetbot/crate@3" async defer>
+  new Crate({
+    server: '804382334370578482',
+    channel: '804383092822900797',
+    defer: false,
+  })
+</script>
+
+
+</body>
+
+</html>
```

Comment:
Please add empty blank line at end of each file.

Reply:
For every file please... In GitHub it shows like this if it's not present.

![image](https://user-images.githubusercontent.com/42013603/224337332-d21b1e57-285f-4f31-bab0-3b371a7d8146.png)